server/src/main/scala/ch.epfl.scala.index.server/DataRepository.scala

```diff
@@ -253,21 +258,17 @@ class DataRepository(
         )
         .should(
           List(
-            termQuery("repository", escaped).boost(1000),
-            fuzzyQuery("repository", escaped),
-            termQuery("artifacts", escaped).boost(20),
-            fuzzyQuery("artifacts", escaped),
-            termQuery("organization", escaped).boost(20),
+            termQuery("repository", escaped).boost(40),
+            fuzzyQuery("repository", escaped).boost(4),
```

Comment:
According to the documentation boost only applies to term queries - https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-boost.html

Reply:
I am not sure if we should include fuzzy query at all - https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-fuzzy-query.html

> The fuzzy query uses similarity based on Levenshtein edit distance.

Reply:
After testing elasticsearch manually, I can confirm that boost applies to fuzzy queries. We are using version 5.4.2  of elasticsearch. It may not be the case anymore on more recent versions, according to the documentation.